recipe/meta.yaml

```diff
@@ -8,28 +8,37 @@ source:
   url: https://github.com/intel/isa-l/archive/v{{ version }}.tar.gz
   sha256: e218b7b2e241cfb8e8b68f54a6e5eed80968cc387c4b1af03708b54e9fb236f1
   patches:
-        # Conda adds some CFLAGS to the environment which are not properly 
-        # filtered by the original nasm-filter.sh
-        # This patch also makes sure the nasm executable from the BUILD_PREFIX is used.
+    # https://github.com/intel/isa-l/commit/1b1ee1e18f5bb4d91e3839405a8557a64a3b2581
+    - patches/fix_erasure_code_wrong_dtype_ppc64le.patch
+    # https://github.com/intel/isa-l/commit/4be96e2437d1aa010978fba54a1b76dfac520961
+    - patches/fix_isal_dflate_icf_finish_for_aarch.patch
+    # https://github.com/intel/isa-l/commit/f1b144bbab7cd1f603565b3b7f92bfb47b86e646
+    - patches/fold_constants_correctly_for_aarch.patch
+    # https://github.com/intel/isa-l/commit/38279f5e9e6455d16b3e97c387f209db56fa994c
+    - patches/avoid_using_X18_register_on_aarch.patch
+    # https://github.com/intel/isa-l/commit/dbaf284e112bea1b90983772a3164e794b923aaf
+    - patches/fix_wasm_spport_on_aarch.patch
+    # Conda adds some CFLAGS to the environment which are not properly
+    # filtered by the original nasm-filter.sh
+    # This patch also makes sure the nasm executable from the BUILD_PREFIX is used.
     - nasm-filter.patch
 
 build:
   number: 1
```

Comment:
Is there a reason you haven't bumped the build number @hmaarrfk @rhpvorderman 

I just got the build update without a build number change and was wondering!

Great work here!

Reply:
Whoops. That was quite a mistake. How best to fix this?

Reply:
Just bump it now, not a problem to forget, don't think it breaks anything, it just could nullify the patch in some cases.

Reply:
Done in #25

Reply:
You'll also want to rebuild any packages you have already built.

Reply:
What do you mean by rebuilding already built packages? Just here in this feedstock? If so how? Or dependents?

Reply:
Your build on 
https://github.com/conda-forge/python-isal-feedstock
https://github.com/conda-forge/python-isal-feedstock/runs/27452755877
just got built with build number 1 with the stricter pinning.

You wiill want to rebuild them for them to be more forgiving for the wrong pin.

Reply:
Build number 1 also has the lenient pinning I think. I believe I changed it before the merge.